tests/documents/test_document_models.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+from django.contrib.contenttypes.models import ContentType
+from django.core.exceptions import ValidationError
+
+from adhocracy4.comments import models as comments_models
+from apps.documents.models import Document
+
+
+@pytest.mark.django_db
+def test_paragraph_save(paragraph):
+    assert '<script>' not in paragraph.text
+
+
+@pytest.mark.django_db
+def test_document_clean(document):
+    another_document = Document(
+        module=document.module,
+        creator=document.creator
+    )
+
+    with pytest.raises(ValidationError):
```

Comment:
where does the exception come from?
i'd suggest to construct valid object in the default factory and either use an explicit InvalidDocumentFactory or construct it directly in the tests context.
i suggest to add this as a new issue, while merging this PR